swan-lake/development-tutorials/observability/overview-of-ballerina-observability.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Overview of Ballerina Observability
+description: See how Ballerina supports observability by exposing itself via metrics, tracing, and logs to external systems.
+keywords: ballerina, observability, metrics, tracing, logs, opentelemetry
+permalink: /learn/overview-of-ballerina-observability/
+active: overview-of-ballerina-observability
+intro: Observability is a measure of how well the internal states of a system can be inferred from the knowledge of its external outputs.
+---
+
+Ballerina Observability is a built-in feature designed to facilitate monitoring and debugging Ballerina applications by collecting, processing, and analyzing performance and operational data. It integrates seamlessly with observability tools and supports three primary pillars of observability: metrics, tracing, and logging.
+
+1. **Metrics** - Provide quantitative insights into the performance of an application, such as request counts, latencies, and error counts.
```

Comment:
aggregated over a period of time is key part of metrics definition. So better reword. Btw, any reason for changing definitions given in https://ballerina.io/learn/observe-ballerina-programs/ first paragraph? I feel those are more concise and accurate.

Reply:
Addressed review suggestions. changed the intro to the existing intro in https://ballerina.io/learn/observe-ballerina-programs/.